src/test/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImportIntegrationTest.java

```diff
@@ -123,21 +146,25 @@ private File runCombineGVCFs(final List<String> inputs, final SimpleInterval int
         for (String input: inputs) {
             args.addArgument("V", input);
         }
-        args.addArgument("L", interval.toString());
+        intervals.forEach(interval -> args.addArgument("L", interval.toString()));
         Arrays.stream(extraArgs).forEach(args::add);
 
         Utils.resetRandomGenerator();
         new Main().instanceMain(makeCommandLineArgs(args.getArgsList(), "CombineGVCFs"));
         return output;
     }
 
-    private void testGenomicsDBAgainstCombineGVCFs(final List<String> vcfInputs, final SimpleInterval interval, final String referenceFile, final String[] CombineGVCFArgs) throws IOException {
+    private void testGenomicsDBAgainstCombineGVCFs(final List<String> vcfInputs, final List<SimpleInterval> intervals,
+                                                   final String referenceFile, final String[] CombineGVCFArgs) throws IOException {
         final String workspace = createTempDir("genomicsdb-tests-").getAbsolutePath() + "/workspace";
 
-        writeToGenomicsDB(vcfInputs, interval, workspace, 0, false, 0, 1);
+        writeToGenomicsDB(vcfInputs, intervals, workspace, 0, false, 0, 1);
         checkJSONFilesAreWritten(workspace);
-        File expectedCombinedVCF = runCombineGVCFs(vcfInputs, interval, referenceFile, CombineGVCFArgs);
-        checkGenomicsDBAgainstExpected(workspace, interval, expectedCombinedVCF.getAbsolutePath(), referenceFile, true);
+        for(SimpleInterval currInterval : intervals) {
+            List<SimpleInterval> tmpList = new ArrayList<SimpleInterval>(Arrays.asList(currInterval));
+            File expectedCombinedVCF = runCombineGVCFs(vcfInputs, tmpList, referenceFile, CombineGVCFArgs);
```

Comment:
Shouldn't you be running `CombineGVCFs` with multiple intervals as well, instead of 1 interval at a time?

Reply:
Related to the earlier question about CombineGVCF